students/paul_c/session03/list_lab.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+# Series 1
+print("\n*** Series 1 ***\n")
+fruits = ["Apples", "Pears", "Oranges", "Peaches"]
+print(fruits)
+user_fruit = input("Please type in a new fruit > ")
+fruits.append(user_fruit)
+print(fruits)
+user_index = input("Please enter a number from 0 - 4 > ")
+user_selected = fruits[int(user_index)]
+print("You have selected fruit number", user_index, '"'+user_selected+'"', "from the list.")
```

Comment:
consider and `f` string for this: `print(f"You have selected fruit number {user_index} {user_selected} from the list.")`  Much easier to read.

Reply:
I need to get a better handle on the use of fstrings. They seem super useful.